Parsers/ASimAuthentication/Parsers/ASimAuthenticationOktaV2.yaml

```diff
@@ -0,0 +1,165 @@
+              ,
+              EventEndTime=TimeGenerated
+              ,
+              EventType=iff(EventOriginalType hassuffix 'start', 'Logon', 'Logoff')       
+              ,
+              TargetSessionId=column_ifexists('ActorSessionId', "")
+              ,
+              TargetUserId= column_ifexists('ActorUserId', "")
+              ,
+              TargetUsername=column_ifexists('ActorUsername', "")
+              ,
+              TargetUserType=column_ifexists('ActorUserType', "")
+              ,
+              TargetUserIdType=column_ifexists('ActorUserIdType', "")
+              ,
+              TargetUsernameType=column_ifexists('ActorUsernameType', "")
+              ,
+              SrcIpAddr = column_ifexists('SrcIpAddr', "")
+              //** extend non-normalized fields to be projected-away 
+              ,
+              ActorDetailEntry,
+              ActorDisplayName,
+              AuthenticationContextAuthenticationProvider,
+              AuthenticationContextAuthenticationStep
+              ,
+              AuthenticationContextCredentialProvider,
+              AuthenticationContextInterface,
+              AuthenticationContextIssuerId,
+              AuthenticationContextIssuerType
+              ,
+              DebugData,
+              DvcAction,
+              OriginalActorAlternateId,
+              OriginalClientDevice,
+              OriginalOutcomeResult,
+              OriginalSeverity,
+              OriginalTarget
+              ,
+              OriginalUserId,
+              OriginalUserType,
+              Request,
+              SecurityContextAsNumber,
+              SecurityContextAsOrg,
+              SecurityContextDomain,
+              SecurityContextIsProxy
+              ,
+              TransactionDetail,
+              TransactionId,
+              TransactionType
+          // ** Aliases
+          | extend 
+              User=TargetUsername
+              ,
+              Dvc=EventVendor
+              ,
+              IpAddr=SrcIpAddr
+          | project-away
+              ActorDetailEntry,
+              ActorDisplayName,
+              AuthenticationContextAuthenticationProvider,
+              AuthenticationContextAuthenticationStep
+              ,
+              AuthenticationContextCredentialProvider,
+              AuthenticationContextInterface,
+              AuthenticationContextIssuerId,
+              AuthenticationContextIssuerType
+              ,
+              DebugData,
+              DvcAction,
+              OriginalActorAlternateId,
+              OriginalClientDevice,
+              OriginalOutcomeResult,
+              OriginalSeverity,
+              OriginalTarget
+              ,
+              OriginalUserId,
+              OriginalUserType,
+              Request,
+              SecurityContextAsNumber,
+              SecurityContextAsOrg,
+              SecurityContextDomain,
+              SecurityContextIsProxy
+              ,
```

Comment:
please re-format. format should look consistent

Reply:
Done